_data/meltano/loaders/target-clickhouse/shaped.yml

```
@@ -0,0 +1,50 @@
+capabilities:
+- about
+- schema-flattening
+- stream-maps
+description: Clickhouse Database
+domain_url: https://www.clickhouse.com/
+executable: target-clickhouse
+keywords:
+- meltano_sdk
+- database
+label: Clickhouse
+logo_url: /assets/logos/loaders/clickhouse.png
+maintenance_status: active
```

```suggestion
maintenance_status: development
```

@BTheunissen how would you categorize the maturity of this target? development, beta, active. Usually development is used to signal that we're working on a new variant but its not necessarily ready to be used, beta suggests that it can be used but might be buggy and active is suggested and ready for prod.

---

I'll go for beta for now as I started on this yesterday afternoon :)